### DISCUSSION TITLE
Feel free to submit ideas

### DISCUSSION THREAD

**Discussion by willwulfken** [willwulfken/MidJourney-Styles-and-Keywords-Reference#2] [Ideas and Style Submissions]
This includes anything that has to do with this repository

- Style ideas
- Artist ideas
- Organizational Ideas
- Any additions or changes
- and more

**Comment by Jerimee** [comment]
I was thinking I might like to contribute. Maybe by adding additional artists?

I have this excellent list of illustrators: https://docs.google.com/spreadsheets/d/1X2g69NDFtH7bGBFIuqwwfS0R_HrqLxxS1nPb5KV8c-o/edit#gid=460486409

Since I moved my request here (sorry if I shouldn't have done that :) let me quote your helpful response here too. Thanks! Look forward to getting you something! :)

> I'll write that down in my to do list so that I can hopefully add some more concise instructions later.
>
> Basically at the moment I'm not too picky. Sending a list of styles is okay with me.
>
> If you send images, just make sure to send the prompt format for confirmation.
> 
> For spheres this should be:
> sphere, style --seed 4776
> 
> For "just the style" this should be:
> style --seed 4776
> 
> You don't need to do them for every version or anything like that. If I want to add those I will, but you can If you'd like.
> 
> If you don't understand the folder structure, categorization, or filename format, I can always take care of that too. Just try to leave a note about that.

**Comment by stefanialasagna** [comment]
It would be amazing to see prompt examples for things like “2 color” or “cutout” in order to get very simplistic images. The goal is to have a single color shape that can be used for crafts like vinyl or laser cutting. 

Kinda related, I tried with v3 to do a “as a sticker” prompt and it worked pretty well to show the outline around the subject as if it looked like it could be peeled off.

**Reply by willwulfken** [reply]
That's an interesting Idea. It would take a little experimenting and would depend on each prompt, but I'm sure there must be something that works at least most of the time.

**Comment by Atlari** [comment]
Would be great to find optimal prompts to manipulate the 7 elements of art:

Line

Color 

Shape

Form

Value

Space

Texture

**Comment by mboox530** [comment]
I'm looking for hunky, early 30s, tattooted, muscled men without shirts, very handsome, full body, romance book cover types. I've been using --ar 6:9 since 6x9" is the typical cover size.  Also would like it on a transparent background 
![image](https://user-images.githubusercontent.com/112494675/190047826-6ada0ba2-c2b6-4acd-9a50-95b558213fea.png)

**Comment by Wildwooddragon** [comment]
Since hands are hard. Could sign language be used as base for learning hand movements and positions?

**Reply by willwulfken** [reply]
Could you explain this a little further please?

**Reply by Atlari** [reply]
I understand something along the lines of:

 since sign language is standardized and broadly documented it would be easier to channel it into specific prompts to achieve more certainty in the desired image.

**Reply by willwulfken** [reply]
Hmm... How exactly are you trying to channel it into prompts?
Create prompts that show sign language or create them by somehow typing in sign language?

**Comment by triniverse** [comment]
I'm sorry if some of these already exists, but here are some examples of Art Styles:
street art
Abstract expressionism
Folk art
traditional art
Pointillism 
Portraiture 
Collage
Trompe-l'œil
Minimalism
Conceptual art
Op art
Grayscale
Hand-drawn animation
Film Noir style
Art Brut
Mixed media
Vector art
ASCII art
Low-poly art
Retro Futurism
Raster art
8-bit Music visualization (yes it´s some kind of art style)
Demoscene art
Glitch art
Non-objective art
Neo-expressionism
Action painting
Automatic drawing
Sound art
Fluxus art movement
Performance art
Still-life painting
Metalworking
Jewelry design
Product photography

**Reply by willwulfken** [reply]
No need to be sorry, I love to see new style ideas and thank you for submitting! I'll take a look at them and see what I can add.

I'm extra curious about 8-bit Music Visualization now 😄

**Reply by willwulfken** [reply]
I took a look at the list and here are the ones that I don't have:

```
Traditional Art
Portraiture
Trompe-L'œil
Conceptual Art
Grayscale
Hand-Drawn Animation
Film Noir Style
Art Brut
Vector Art
ASCII Art
Low-Poly Art
Raster Art
8-Bit Music Visualization
Demoscene Art
Glitch Art
Non-Objective Art
Neo-Expressionism
Action Painting
Automatic Drawing
Sound Art
Fluxus Art Movement
Performance Art
Still-Life Painting
Metalworking
Jewelry Design
Product Photography
```

There are a lot of good ones in here that I completely missed so thank you very much!

**Reply by triniverse** [reply]
Nice to hear, I´m glad to help.
Maybe I´ll add some more soon. 
Actually I got all of these from ChatGPT, when I asked for suggestions of different art styles.

**Comment by castelcerf** [comment]
I i feel that "diagramatic" give better result than just "diagram"

And I dont know if you already get the
"stained glass"

**Reply by willwulfken** [reply]
Interesting Insight. Thank you for mentioning.

**Comment by PaladinCiel** [comment]
I think various styles of horror/fantasy/macabre are missing. to include Dark Fantasy.

Dark Fantasy in particular is a point of contention for me. Months back it used to give me some great looking werewolf figures, but after all the AI studios went and started purging the likes of Greg Rutkowski it left those old prompts of mine no longer capable of recreating the effect no matter what I've tried. 

What I used to be able to get...
https://cdn.discordapp.com/attachments/1039689995411267677/1048011494090084423/Ciel_Longshot_hyper_realistic_Horror_styled_dark_fantasy_ornate_2f151ac7-e88f-4535-a00c-aeeee56842ea.png

After the changes...
https://cdn.discordapp.com/attachments/1039689995411267677/1071162234186772632/Ciel_Longshot_hyper_realistic_Horror_styled_dark_fantasy_ornate_c7a0fd42-b5f7-40b6-b175-756433365edf.png

Beyond that here's a list of some styles and names I commonly try to find use for in my prompts that I've been trying to find more to compliment. I've had a hard time knowing which styles are actually doing anything of value and being more directly able to compare them would be a big help.

Horror styled, 
Vampirella styled,
Dark Fantasy,
HorrorBooru, 
Horrorism,
Gothic Horror,
Cosmic Horror,
Junji Ito,   
80s horror style, (20s, 30s, 40s, 50s, 60s, 70s, ??)
Horror Anime, 
Vampire Hunter D styled, 
Bloodborne styled,
Castlevania Styled,
Go Nagai,
Macabre,
Spooky,
Creepy,
Haunting, 
80s styled horror movie,

been wanting to find styles that can mimic the realistic latex made horror effects of the 80s

I've seen that the system can render very realistic werewolves at times but never when I seem to want it, I cant pin down the styles its using in those rare occasions.

**Reply by willwulfken** [reply]
Thank you!

**Comment by sugarchrist** [comment]
I would like to see "website" prompt
Thanks'

**Reply by willwulfken** [reply]
Here you go: [[🖥 Digital] > [🖥🌐 Websites]](https://github.com/willwulfken/MidJourney-Styles-and-Keywords-Reference/blob/main/Pages/MJ_V4/Style_Pages/Just_The_Style/Digital.md)

**Comment by IsaCreative** [comment]
Hello i have watched and read many 'consistent character' creation how to's but wondered if this is something you had tested rigourosly. If so it would without a doubt be an extremley valuable resource.

**Comment by diesektion** [comment]
hey there,

first of all: What a great page. Thank you so much for making my first steps with midjourney so much easier. 

i would like to contribute "tiltshift" for the category "Zoom, Pan, and Tilt" in "Camera & Lenses"

**Reply by willwulfken** [reply]
Thank you for your idea! I've added tiltshift
[[📷 Camera, Film, and Lenses]](https://github.com/willwulfken/MidJourney-Styles-and-Keywords-Reference/blob/main/Pages/MJ_V4/Style_Pages/Just_The_Style/Camera.md)
[[📷 Camera, Film, and Lenses] [🌳 Tree]](https://github.com/willwulfken/MidJourney-Styles-and-Keywords-Reference/blob/main/Pages/MJ_V4/Style_Pages/Tree/Camera.md)

**Reply by diesektion** [reply]
great. Thanks!

**Comment by BABA433** [comment]
Do you guys wonder how to find best tailored made Midjourney prompts?
Don't worry I made full library of FREE Prompts check out here - [FREE Midjourney Prompts](https://www.freemidjourneyprompt.com/)

**Comment by Chakhdz** [comment]
I have a list of photographers and cinema directors

**Comment by luglio13** [comment]
Presenting the picture of Spanish streets in the 1900s with elements of mystery and thriller, the artwork is in the style of Gaudí, portraying the essence of this great architect's work, and uses a grey color tone.
